.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.18
+          go-version: 1.20
```

Comment:
not `1.19`?

Reply:
go version here doesn't not matter since actual binary is built in docker.  That said, I agree that this is confusing, -- changed it to 1.19 too.